### PR TITLE
Add mugdha273

### PR DESCRIPTION
**Pull request by mugdha273**
This pr adds mugdha273.txt file in _post folder which carries details of name, username, and learning point.

### CODE REVIEW

File: _post/sample.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Name: <yourname>
 Github username: <github-username> 
-What you learned today: <what your learned in today's talk>
```
Comment (nlok5923) [inline _post/sample.txt:3]: revert unnecesary changes

**Review by nlok5923 — CHANGES_REQUESTED** [review]
Thanks PTAL

**Comment by nlok5923** [comment]
also add proper pr description of what changes this pr brings.

File: _post/sample.txt (hunk 1 above)
Comment (mugdha273) [reply _post/sample.txt:3]: done

File: _post/sample.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
 Name: <yourname>
```
Comment (nlok5923) [inline _post/sample.txt:2]: remove the extra new line

**Review by nlok5923 — CHANGES_REQUESTED** [review]
thanks PTAL

**Comment by nlok5923** [comment]
> also add proper pr description of what changes this pr brings.

.

**Comment by mugdha273** [comment]
> > also add proper pr description of what changes this pr brings.
> done

**Comment by nlok5923** [comment]
LGTM!! thanks 👍

**Pull request merged**